pkg/inspector/image-inspector.go

```diff
@@ -39,6 +40,7 @@ const (
 	OPENSCAP_URL_PATH  = API_URL_PREFIX + "/" + VERSION_TAG + "/openscap"
 	CHROOT_SERVE_PATH  = "/"
 	OSCAP_CVE_DIR      = "/tmp"
+	PULL_LOG_INTERVAL  = 10
```

Comment:
let's make the name descriptive of the time unit that will be used or add a comment noting it.